### PR TITLE
Require scramblers signatures

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
Addressing #686

### CODE REVIEW

**Comment by zeecho** [comment]
Since it's becoming a _must_, what should we do if a scrambler signature is missing?

**Comment by Ivan-Brigidano** [comment]
> Since it's becoming a must, what should we do if a scrambler signature is missing?

@zeecho this is something the WRC is debating. Please keep in mind that this PR is a draft so it not definitely

**Comment by EdHollingdale** [comment]
As well as addressing the obvious (what should we do if a scramble signature is missing), has any analysis been done as to whether scramble signatures are effective? We've been using them for a year - have they reduced misscrambles and duplicates? I can say anecdotally the answer is no. Is it worth the time cost and potential for unnecessary punishing of the competitor for little to no benefit?

**Comment by Ivan-Brigidano** [comment]
@EdHollingdale I feel that it isn't used in some competitors. However from personal experience I have noticed less misscrambles in competitions that used them. In addition in some reports it helped us to determine who makes the misscrambles and make a better decision.

This could also help in #851 as you said:

> Both of these can be fixed far better by encouraging a scramble checker - which also solves additional problems.

**Comment by Laura-O** [comment]
In general, it's impossible to say whether scrambler signatures reduce misscrambles. As long as we do not record each official solve and check the scramble, we can only speak about *reported* misscrambles.
We have been using scrambler signatures for a long time now and one positive aspect is that scramblers feel more responsible for doing a good job. I do not know if this really has a positive effect on how they actually perform though.

Apart from that, Lucas [recently mentioned](https://docs.google.com/document/d/1PKIN5wOqbunIeNDQVyQH74BdahLGLimk6KZcaa1CT1U/edit?disco=AAAACx6sdo4) that the signatures should be called "scramble signature" as scrambles can also be signed by a designated scramble checker. This PR only mentions scramblers.

File: wca-regulations.md [hunk 1]
```diff
@@ -365,7 +365,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A2c) After the scrambler starts scrambling the puzzle, the competitor must not see the puzzle until the inspection phase starts.
         - A2c1) The scrambler places a cover over the scrambled puzzle that makes it impossible for any competitors or spectators to see any part of the puzzle. The cover remains over the puzzle until the start of the attempt.
     - A2d) When taking a puzzle from the scrambler, the judge briefly inspects the puzzle to ensure thorough scrambling of the puzzle. The judge raises any concerns with the scrambler, who then conducts a detailed check.
-        - A2d1) The organization team should require scramblers to sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image.
+        - A2d1) The scrambler must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image.
```

Comment (Ivan-Brigidano) [reply wca-regulations.md]: Currently regulations don't recognize "scramblers checkers". So I guess judge would be te role that fits better here

**Comment by julesDesjardin** [comment]
In addition, I'm not sure whether it actually make people more aware of their mistakes. Signing is now just another automatic thing to do, before putting the puzzle in the cover. Careless scramblers who don't check scrambles (or check them very fast) continue to do so, simply signing before. In my opinion, to reduce misscrambles we should instruct scramblers to check more often or more carefully. When I do that I'm usually met with "pfff, I make a good job, look, this one is good". Using good scramble checkers and showing them their mistakes directly after they scrambled is the only way to make them learn.

In addition, some misscrambles happen during the cube transport (the famous Max Park's OH "WR" is a probable case, and it also happened to me with 100% certainty), and we can't really do anything against it.

However knowing who scrambled what, and which scramble, is pretty useful (maybe more than judge signatures sometimes)

**Comment by xsrvmy** [comment]
Slight nuance , but I think "the scrambler" should be changed to "a scrambler" in case the original scrambler forgets to sign, and when the cube is sent back for signing no one remembers who scrambled.

**Comment by xsrvmy** [comment]
Another issue is regarding to 6 7 and mega. What should happen if the scramble is incorrect? And what happenes if the puzzle is sent back and the scramble is incorrect?

File: wca-regulations.md [hunk 2]
```diff
@@ -365,7 +365,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
         - A2c1) The scrambler places a cover over the scrambled puzzle that makes it impossible for any competitors or spectators to see any part of the puzzle. The cover remains over the puzzle until the start of the attempt.
     - A2d) When taking a puzzle from the scrambler, the judge briefly inspects the puzzle to ensure thorough scrambling of the puzzle. The judge raises any concerns with the scrambler, who then conducts a detailed check.
-        - A2d1) The organization team should require scramblers to sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image.
+        - A2d1) The scrambler must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)).
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)).
```

File: wca-regulations.md [hunk 3]
```diff
@@ -365,7 +365,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
         - A2c1) The scrambler places a cover over the scrambled puzzle that makes it impossible for any competitors or spectators to see any part of the puzzle. The cover remains over the puzzle until the start of the attempt.
     - A2d) When taking a puzzle from the scrambler, the judge briefly inspects the puzzle to ensure thorough scrambling of the puzzle. The judge raises any concerns with the scrambler, who then conducts a detailed check.
-        - A2d1) The organization team should require scramblers to sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image.
+        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)).
```
Comment (lgarron) [inline wca-regulations.md]:
```suggestion
        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler may sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)).
```

**Review by xsrvmy — COMMENTED** [review]
I still think "the scrambler" should be replaced with "a scrambler".
Also is an authorized judge supposed to refer to judges or any person?

**Review by lgarron — APPROVED** [review]
Based on concerns about how to handle missing signatures, #883 adds that missing signatures are okay if they happen occasionally.

**Pull request merged**